test_cpp/tests/attach/attach_test.cpp

```diff
@@ -58,7 +58,7 @@ class AttachTest {
         nc_attacher_set_app_info(&attach_, appName_, appVersion_);
         nc_attacher_set_device_info(&attach_, productId_.c_str(), deviceId_.c_str());
         // set timeout to approximately one seconds for the dtls handshake
-        nc_attacher_set_handshake_timeout(&attach_, 50, 500);
+        nc_attacher_set_handshake_timeout(&attach_, 1000, 1000);
```

Comment:
keep timeout small for mbedtls and keep the values different so timeouts are triggered